src/tools/driver/cmd_line_utils.dart

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) 2014, the Fletch project authors. Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE.md file.
+
+library driver.cmd_line_utils;
+
+int getPortNumberFromArg(String arg, String helpText) {
```

Comment:
Will address in follow on. I agree it's much nicer not having this helper method having UI interactions.